src/advanced-options.tsx

```diff
@@ -15,16 +15,20 @@ const AdvancedOptions = (
 
   const trans = useTranslator('jupyterlab');
 
-  const handleInputChange = (e: ChangeEvent<HTMLInputElement>) =>
+  const handleInputChange = (e: ChangeEvent<HTMLInputElement>) => {
+    if (props.jobsView !== JobsView.CreateForm) {
```

Comment:
Could be addressed later: `jobsView` and `JobsView` should be renamed, as they might refer to a singular job or one or more job definitions

Reply:
Let's track this in a separate issue. https://github.com/jupyter-server/jupyter-scheduler/issues/210